<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>301</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normaloff>../../../../../Work/QT/NewTest/favicon.ico</normaloff>../../../../../Work/QT/NewTest/favicon.ico</iconset>
       </attribute>
       <attribute name="label">
        <string>列表项操作</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>3</y>
          <width>231</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowminimumheight="1,1,1,1,1">
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButton_Listini">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButton_ListClear">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="toolButton_ListInsert">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QToolButton" name="toolButton_ListAppend">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QToolButton" name="toolButton_ListDelete">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>301</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normaloff>../../../../../Work/QT/NewTest/menu-ico/About.png</normaloff>../../../../../Work/QT/NewTest/menu-ico/About.png</iconset>
       </attribute>
       <attribute name="label">
        <string>列表排序</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>241</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,2,2,1">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_ListSort">
           <property name="text">
            <string>允许排序</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButton_ListSortUporder">
           <property name="text">
            <string>升序排序</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="toolButton_ListSortDownorder">
           <property name="text">
            <string>降序排序</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>58</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>301</width>
         <height>305</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normaloff>../../../../../Work/QT/NewTest/menu-ico/Check.png</normaloff>../../../../../Work/QT/NewTest/menu-ico/Check.png</iconset>
       </attribute>
       <attribute name="label">
        <string>信号发射时机</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>11</y>
          <width>281</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_TextClear">
             <property name="text">
              <string>清空文本</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_TextAddLine">
             <property name="text">
              <string>添加空行</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>当前项变化</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Item">
           <property name="text">
            <string>项可编缉</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,1,1">
         <item>
          <widget class="QToolButton" name="toolButton_SelectItem">
           <property name="text">
            <string/>
           </property>
           <property name="popupMode">
            <enum>QToolButton::MenuButtonPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_SeletcAll">
           <property name="text">
            <string>...</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_SelectNone">
           <property name="text">
            <string>...</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_SelectReverse">
           <property name="text">
            <string>...</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item2</string>
          </property>
          <property name="font">
           <font>
            <italic>false</italic>
           </font>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionList_Ini"/>
   <addaction name="actionList_Clear"/>
   <addaction name="actionList_Insert"/>
   <addaction name="actionList_Append"/>
   <addaction name="actionList_Delete"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionList_Ini">
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionList_Clear">
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actionList_Insert">
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionList_Append">
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionList_Delete">
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionList_SletectAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actionList_SelectNone">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
  </action>
  <action name="actionList_SelectReverse">
   <property name="text">
    <string>返选</string>
   </property>
   <property name="toolTip">
    <string>返选</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSelectPopMenu">
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionSelectPopMenu</sender>
   <signal>triggered()</signal>
   <receiver>actionList_SelectReverse</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
